tests/scripts/run_models.sh

```diff
@@ -18,10 +18,8 @@ if [[ $ARCH_NAME == "wormhole" ]]; then
 fi
 
 if [[ $ARCH_NAME == "wormhole_b0" ]]; then
-  env pytest models/experimental/mamba/tests/test_full_model.py
-  env pytest models/experimental/mamba/tests_opt/test_full_model.py
-  env pytest models/experimental/mamba/tests/test_benchmarks.py
-  env pytest models/experimental/mamba/tests/test_demo.py
+  env pytest models/demos/mamba/tests/test_full_model.py::test_mamba_inference_ci
```

Comment:
I think we should run everything in our test folder except for perf. I think you could clean up those extra functions in `test_full_model.py` and then just list all of the test files we have (minus perf) here. 

What might be even better - rename `test_mamba_perf` to `run_mamba_perf` and we can just list `pytest models/demos/mamba/tests/test_*` here. It would be nice to avoid listing these things explicitly so when we add new test files they go into CI automatically.

Reply:
Added more unit tests in nightly.